web/satellite/tests/unit/notifications/Notification.spec.ts

```diff
@@ -1,9 +1,27 @@
 // Copyright (C) 2019 Storj Labs, Inc.
 // See LICENSE for copying information.
 
-import { mount, shallowMount } from '@vue/test-utils';
+import { createLocalVue, mount, shallowMount } from '@vue/test-utils';
+import Vuex from 'vuex';
+import sinon from 'sinon';
 import Notification from '@/components/notifications/Notification.vue';
 import { NOTIFICATION_TYPES } from '@/utils/constants/notification';
+import { DelayedNotification } from '@/types/DelayedNotification';
+import { makeNotificationsModule } from '@/store/modules/notifications';
+import { NOTIFICATION_ACTIONS } from '@/utils/constants/actionNames';
+
+const localVue = createLocalVue();
+
+localVue.use(Vuex);
+const pauseSpy = sinon.spy();
```

Comment:
you do not need this spy

just use in your tests smth like

jest.spyOn(notificationModule.actions, NOTIFICATION_ACTIONS.PAUSE)...